app/static/front-end-scripts/login.js

```diff
@@ -1,23 +1,31 @@
-$("document").ready(function() {
+let buttonClicked = "";
+
+$("document").ready(function () {
 	$("#loginForm").submit(function (e) {
 		$("#loginErrorMsg").text("");
 		e.preventDefault();
 		let form = document.getElementById("loginForm");
 		//gets data entered in the form and converts it to json
 		let formData = Object.fromEntries(new FormData(form));
 		let request = $.ajax({
-			url: form.dataset.link,
+			//The id of the submit buttons is the route
+			url: buttonClicked,
 			type: "POST",
 			data: formData,
 			dataType: "text",
-		}).done(function(msg) {
+		}).done(function (msg) {
 			location.reload(true)
-		}).fail(function(msg) {
+		}).fail(function (msg) {
 			if (msg.status >= 300) {
 				$("#loginErrorMsg").text(JSON.parse(msg.responseText).reason);
 			}
 		});
 	});
+
+	$("#loginForm input[type = 'submit']").click(function (e) {
```

Comment:
What is the point of the e parameter here?

Reply:
Additionally, I'm not certain of best practice here, but i don't particularly like this selector. I think it would be better with a new class instead of the id and input.

Reply:
Good catch on the e - it isnt used dunno why i put it there (likely a carryover from other attempts)
I'll cede your second point, but I personally prefer it this way as it removes code repetition